### PR TITLE
fix(rules/header): trim lead/trail whitespace

### LINKED ISSUE #8: Should handle trailing whitespace in license.js
__Describe the Bug__

Currently, trailing newlines are not allowed in the license.js. However,
many editors make it difficult to remove trailing newlines.

__Steps to Reproduce__

1. follow the user guide
2. introduce a newline at the end of the license.js file

__Expected Behavior__

Expected the license to work. However the license.js file and all source are considered invalid

__Environment__

n/a

### PR DESCRIPTION
**Pull request by srl295**
Related to #5, use trim() so that trailing whitespace is corrected before test.

Closes #8

### CODE REVIEW

**Comment by nikku** [comment]
Happy to consider this if you add a meaningful test case, verifying what you contribute works.

**Comment by srl295** [comment]
> Happy to consider this if you add a meaningful test case, verifying what you contribute works.

Great, will do

**Comment by srl295** [comment]
@nikku updated to include test case that fails w/o change, and also updated docs.

File: tests/fixtures/license-header-blank-line-at-end.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Copyright (c) Foo Corp.
+ *
+ * This source code is licensed under the WTFPL license found in the
+ * LICENSE file in the root of this projects source tree.
+ */
```
Comment (srl295) [inline tests/fixtures/license-header-blank-line-at-end.js:6]: newline at end here

**Comment by nikku** [comment]
Thanks for your quick response. Merged with some minor adjustments via https://github.com/nikku/eslint-plugin-license-header/pull/10.

**Comment by srl295** [comment]
@nikku welcome. Thank you for this plugin!

**Pull request closed without merging**